src/index.ts

```diff
@@ -13,12 +13,13 @@ process.on('unhandledRejection', (error) => {
 	throw error
 })
 
-const devcardURL = (user_id: string): string =>
-	`https://api.daily.dev/devcards/v2/${user_id}.png?r=${new Date().valueOf()}&ref=action`
+const devcardURL = (user_id: string, type = 'default'): string =>
+	`https://api.daily.dev/devcards/v2/${user_id}.png?type=${type}&r=${new Date().valueOf()}&ref=action`
 
 ;(async function () {
 	try {
 		const user_id = core.getInput('user_id')
+		const type = core.getInput('type')
```

Comment:
Please do a validation check that the type is either `default` or `wide`.

Reply:
I will update. Thank you for comment.

Reply:
Maybe even use an enum type for the `type` parameter. Or string union type. E.g. something like:

```typescript
// enum
enum DevCardType {
  Vertical = 'default',
  Horizontal = 'wide',
  Twitter = 'x',
}

// union
type DevCardType = 'default' | 'wide' | 'x';
```

You can then use the `DevCardType` for the validation as well.

Reply:
Hi @omBratteng, I just updated the code. Only change the `src/index.ts` and `action.yml` files, as well as check the validity of the `type`.